src/Network/Probe.php

```diff
@@ -778,6 +783,12 @@ private static function pollNoscrape($noscrape_url, $data)
 			$data["poll"] = $json["dfrn-poll"];
 		}
 
+		if (isset($json["hide"])) {
+			$data["hide"] = (bool)$json["hide"];
```

Comment:
Are you sure `hide` can't be the strings `"true"` or `"false"` ?

Reply:
It is generated here: https://github.com/friendica/friendica/blob/develop/mod/noscrape.php#L47

Means: I'm totally sure that it isn't a string.

Reply:
The other problem is that the array key may not be existing. Please initialize it as `false` in the above `$json` array declaration.

Reply:
I'm not sure where I should do this? In ```Probe.php``` I'm checking with ```isset```, so everything is fine, isn't it?

Reply:
Ah, yeah, I forgot about that. Good point.